components/datePicker/datePickerContainer.tsx

```diff
@@ -24,9 +22,6 @@ export const DatePickerContainer: FC<DatePickerContainerProps> = ({ error, label
   const router = useRouter();
 
   // TODO: Move locale setting in global file
```

Comment:
You should delete a `TODO` reminder if the corresponding function is implemented.

Reply:
done